python-sdk/src/astro/options.py

```diff
@@ -5,3 +5,12 @@
 class LoadOptions:
     def empty(self):
         return NotImplementedError()
+
+
+@attr.define
+class SnowflakeLoadOptions(LoadOptions):
+    file_options: dict = attr.field(init=True, factory=dict)
+    copy_options: dict = attr.field(init=True, factory=dict)
+
+    def empty(self):
+        return not self.file_options and not self.copy_options
```

Comment:
Same here. I think it is important to cover this.

Reply:
added test